paddle/fluid/framework/version.h

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
should we unify generated `version.py` to use a same c++ implement? The `version.py` is generated by `setup.py.in`, and to use it, call `print paddle.version.show()`

Reply:
Done